create_jenkins_job.py

```diff
@@ -279,7 +277,7 @@ def create_job(os_name, job_name, template_file, additional_dict):
                 # We do not want to test more than this as we observe issues with the clang libcxx standard library
                 # we don't plan to tackle for now. The important part of this nightly is to make sure the code compiles
                 # without emitting thread-safety related warnings.
-                'test_args_default': '--event-handlers console_direct+ --executor sequential --packages-select rcutils',
+                'test_args_default': data['test_args_default'].replace(' --retest-until-pass 2', '') + ' --packages-select rcutils'
```

Comment:
Is omitting the retest arg related to the rest of the PR somehow?

Reply:
Oh I see. Because we're reusing the defaults we have to tweak them to match the previous test args. Rescinded.

Reply:
The patch doesn't change the actual behavior (beside skipping `xfail` of course). Before all args where repeated and the retest arg omitted as described in the comment above. After it uses the previous definition of the args and manipulate it accordingly to avoid duplication.